apps/user-portal/src/components/settings-section.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import * as React from "react";
+import { Button, Card } from "semantic-ui-react";
+
+/**
+ * Proptypes for the settings section component.
+ */
+interface ComponentProps {
+    header: string;
+    description: string;
+    actionTitle: string;
+    isEdit: boolean;
+    onClick: any;
```

Comment:
Make it a function type instead `any`

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/58/commits/07ca92c8bee410cb2df318d1010aef36700012a7